File: verdict_test/ce/submission.go
package main

import (
	"fmt",
	"bufio",
)

func main() {
	input := bufio.NewReader(os.Stdin)
	output := bufio.NewWriter(os.Stdout)

	vr n i
	fmt.Fscan(input, &n)

	a := make([]int, n)

	for i := 0; i < n; i++ {
		fmt.Fscan(input, &a[i])
	}

	var m int
	m = 0

	var maxval int
	maxval = a.max()

	for i := 0; i < n; i++ {
		if a[i] == maxval {
			m = i
			break
		}
	}

	fmt.Fprintln(output, m)
}

File: verdict_test/tle/submission.js
const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});

rl.on("line", (input) => {
    const n = parseInt(input);
    rl.close();
    while (true) {
        console.log(n);
    }
});
